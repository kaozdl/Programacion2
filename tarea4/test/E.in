cola_prioridad 1 (10,z) (30,y) (17,x) (99,no) (98,no) (1,si) .
cola_prioridad 99 .
cola_prioridad 5 .
cola_prioridad 4 .
cola_prioridad 3 .
cola_prioridad 2 .
cola_prioridad 1 .
cola_prioridad 1 .
Fin
