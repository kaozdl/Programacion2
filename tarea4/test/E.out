1>
(1,si)
2>(99,no)
(10,z)
3>
(17,x)
4>
(30,y)
5>
(98,no)
6>
(99,no)
7>
8>
9>Fin.
